tests/framework/ab_test.py

```diff
@@ -165,11 +165,14 @@ def git_ab_test_guest_command(
     paths to firecracker and jailer binaries."""
 
     @with_filelock
```

Comment:
One thing to note here that I forgot is that we took the look for both building **and** running the test. The 2nd part is unnecessary, so now we only take the lock *if* the binary does not exist, and only for that part.